internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -1152,6 +1169,16 @@ func upgradeSettingsSchema() *pluginsdk.Schema {
 					Optional: true,
 					Default:  "10%",
 				},
+				"drain_timeout_in_minutes": {
+					Type:     pluginsdk.TypeInt,
+					Optional: true,
+					Default:  30,
```

Comment:
Could we add a TODO to add this default to the docs for 4.0?

Reply:
I am going to remove this since I found this did not work as expected when I did some 4.0 testing. I will open a follow up PR next week that will involve some 4.0 fixes for AKS, including a behavioural change to the `upgrade_settings` block